x-pack/legacy/plugins/canvas/shareable_runtime/test/context_jest.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+jest.mock('../supported_renderers');
```

Comment:
This seems like a random place to put this mock.

Reply:
It was because anything using this test context needed the renderers mocked... rather than put it in every file, I put it in this context.  I can add it to every test file, or add a comment, whichever you'd recommend.